backend/games/word_guesser/WordGuesser.py

```diff
@@ -0,0 +1,124 @@
+from enum import IntEnum
+from random import randint
+from typing import Dict, List
+
+MAX_GUESSES = 6
+
+# initialize valid guesses and possible solutions
+SOLUTIONS: str
+VALID_WORDS: str
+
+try:
+    solutions_file = open(
+        "backend/games/word_guesser/WordGuesser_Solutions.txt"
+    )
+    guesses_file = open(
+        "backend/games/word_guesser/WordGuesser_ValidGuesses.txt"
+    )
+    SOLUTIONS = solutions_file.read()
+    SOLUTION_LIST = SOLUTIONS.splitlines()
+
+    VALID_WORDS = guesses_file.read()
+    VALID_WORDS_LIST = VALID_WORDS.splitlines()
+except:
+    print("Files not readable. Check file or file path.")
+
+
+class CorrectnessScore(IntEnum):
+    correct = 3
+    in_word = 2
+    not_in_word = 1
+
+
+class WordGuesser:
+    def __init__(self, goal=None):
+        self.guesses = {"guess": "", "results": []}
+        self.game_board: List[str] = []
+        self.progress: Dict = {}
+        self.goal_word = goal
+        self.guess_count = 0
+
+    def clear_data(self):
+        self.guesses = {"guess": "", "results": []}
+        self.game_board: List[str] = []
+        self.progress: Dict = {}
```

Comment:
Can you change this to a list so that we can track the order of guesses?

Reply:
would a list of tuples be a good replacement for a dict?

Reply:
Yeah you could do a list of tuples, where the tuples are `(guess, result)` or you can just do a list of the result dictionaries.